src/device-state.ts

```diff
@@ -45,7 +45,7 @@ import {
 	AppState,
 } from './types';
 import * as dbFormat from './device-state/db-format';
-import * as apiKeys from './lib/api-keys';
+import * as apiKeys from './device-api/api-keys';
```

Comment:
Can we try to export the specific module that `device-state` (and maybe others) on `device-api/index.ts`, I would love if we can get to a point where modules can work more like black boxes, so only what is exported can be used. 

In this case the import would change to

```
import {initialised as initialiseApiKeys, cloudApiKey} from './device-api'
``` 

Also, it is probably not necessary in this PR, but maybe the API doesn't need to be initialised here, but the initialised method can be called when the cloudApiKey is obtained. Let's say we make `cloudApiKey` a function `getGlobalApiKey`

That function can be implemented like this

```
export async function getGlobalApiKey() { 
    await initialised();
    return generateCloudKey();
}
```

And only `getGlobalApiKey` needs to be exported by `device-api` from this module.

Reply:
I added 2 new commits in https://github.com/balena-os/balena-supervisor/commit/a105417e89d4a33282e313ebb87bd0eaef6e4761 to address this. The dbus commit was because using deviceApi directly instead of apiKeys in compose/utils broke the unit tests for it.

Reply:
Additionally the dbus commit is just better overall to decomplicate the test environment, and makes importing lib/dbus side-effect-free.

Reply:
Yeah, I think this works in line with the latest changes. Some day we'll move to a dependency injection pattern at which point I hope these sort of singleton type initializations won't be necessary anymore 🙏